src/ocamlorg_frontend/components/learn_components.eml

```diff
@@ -154,31 +154,46 @@ let right_books_content_block books ~title ~(link : link) =
   </div>
 
 let contribute_footer ~href =
-  <div class="pt-10 mt-16 border-t border-slate-200 lg:grid lg:grid-cols-3 gap-4 text-slate-500">
-    <div class="lg:col-span-1">
-      <h2 class="mt-0 mb-4 text-base font-semibold leading-6 lg:text-base">Help Improve Our Documentation</h2>
-      <p class="text-sm mt-0 mb-4 max-w-xs">All OCaml docs are open source. See something that's wrong or unclear? Submit a pull request.</p>
-      <a
-        class="inline-block relative py-1 px-4 font-medium no-underline align-middle bg-transparent rounded-md border border-solid appearance-none cursor-pointer select-none whitespace-no-wrap hover:no-underline"
-        href="<%s href %>"
-      >
-        <%s! Icons.github_pull_request "inline-block overflow-visible mr-0 align-text-bottom select-none" %>
-        Contribute</a
-      >
+  <div class="my-5 border-y border-slate-400 pt-5">
+    <div class="mb-10 mt-4">
+      <h2 class="mb-4 mt-0 text-lg font-bold leading-6">Also Read:</h2>
+      <ul class="list-disc marker:text-orange-500">
+        <li class="mb-1">
+          <a href="https://github.com/ocaml/ocaml.org/" class="cursor-pointer bg-transparent no-underline hover:underline"> Title of article or tutorials one related to above lorem ipsum dolorsed do eiusmod </a>
+        </li>
+        <li class="mb-1">
+          <a href="https://github.com/ocaml/ocaml.org/" class="cursor-pointer bg-transparent no-underline hover:underline"> Title of article or tutorials one related to above lorem ipsum dolorsed do eiusmod </a>
+        </li>
+        <li class="mb-1">
+          <a href="https://github.com/ocaml/ocaml.org/" class="cursor-pointer bg-transparent no-underline hover:underline"> Title of article or tutorials one related to above lorem ipsum dolorsed do eiusmod </a>
+        </li>
+      </ul>
     </div>
-    <div class="mt-4 lg:mt-0 lg:col-span-1 lg:col-start-3">
-      <h2 class="mt-0 mb-4 text-base font-semibold leading-6 lg:text-base">Still need help?</h2>
-      <div class="text-sm mb-2">
-        <a href="https://discuss.ocaml.org/" class="font-semibold no-underline bg-transparent cursor-pointer hover:underline">
-          <%s! Icons.community "inline h-4 w-4" %>
-          Ask the OCaml Community</a
-        >
+    <div class="grid grid-cols-2 gap-4 border-t border-slate-400 px-6">
+      <div class="border-r border-slate-400 pb-3">
+        <h2 class="mb-2 text-lg font-bold leading-6">Still need help?</h2>
+        <div class="mb-2 text-sm ">
+          <a href="https://discuss.ocaml.org/" class="font-normal text-lg no-underline bg-transparent cursor-pointer hover:underline">
+            <%s! Icons.user_group "inline h-5 w-5" %>
+            Ask the OCaml Community</a
+          >
+        </div>
+        <div class="mb-2 text-sm">
+          <a href="https://github.com/ocaml/ocaml.org/issues/new" class="font-normal text-lg no-underline bg-transparent cursor-pointer hover:underline">
```

Comment:
For some reason, I am having trouble here with some of the icon being cut off. Also, the text besides the icons is not aligned between the two lines.

![Screenshot from 2023-10-12 13-43-20](https://github.com/ocaml/ocaml.org/assets/6594573/37b66950-4d8d-4a2b-8a10-b5ca6b4a6f46)

I would recommend changing the links here to use a `flex` layout with a `gap` property and `items-center`, instead of going for inline icon.